beacon-chain/core/electra/effective_balance_updates.go

```diff
@@ -24,6 +31,35 @@ import "github.com/prysmaticlabs/prysm/v5/beacon-chain/state"
 //	        ):
 //	            validator.effective_balance = min(balance - balance % EFFECTIVE_BALANCE_INCREMENT, EFFECTIVE_BALANCE_LIMIT)
 func ProcessEffectiveBalanceUpdates(state state.BeaconState) error {
```

Comment:
I think you need to update the electra.ProcessEpoch function to include this one